CladeInfo.csv

```diff
@@ -1,31 +1,31 @@
-clade,maxN,outgroup,outgroupLoc
-B1-11,56000,AFT-21-06837-21,s3://s3-csu-003/v3-2/Results_NB501786_0361_24Jun22/
-B2-11,55500,AF-12-01357-20,s3://s3-csu-003/v3-2/Results_10287_27Jun22/
-B3-11,52500,AFT-21-06837-21,s3://s3-csu-003/v3-2/Results_NB501786_0361_24Jun22/
-B4-11,54000,AF-61-00379-18,s3://s3-csu-003/v3-2/Results_10102_28Jun22/
-B5-11,56000,AF-12-01357-20,s3://s3-csu-003/v3-2/Results_10287_27Jun22/
-B6-11,60000,AF-61-03263-17,s3://s3-csu-003/v3-2/Results_10062_27Jun22/
-B6-12,53000,AF-21-06026-18,s3://s3-csu-003/v3-2/Results_10100_28Jun22/
-B6-13,53000,AF-61-03263-17,s3://s3-csu-003/v3-2/Results_10062_27Jun22/
-B6-14,53000,AF-21-06026-18,s3://s3-csu-003/v3-2/Results_10100_28Jun22/
-B6-15,52000,AFT-61-00566-22,s3://s3-csu-003/v3-2/Results_NB552234_0122_24Jun22/
-B6-16,52000,AFT-21-05881-21,s3://s3-csu-003/v3-2/Results_NB552234_0069_24Jun22/
-B6-21,54000,AF-61-00479-18,s3://s3-csu-003/v3-2/Results_10101_28Jun22/
-B6-22,52000,AF-21-03418-15,s3://s3-csu-003/v3-2/Results_10212_25Jun22/
-B6-23,60000,AF-21-03418-15,s3://s3-csu-003/v3-2/Results_10212_25Jun22/
-B6-31,52000,AF-21-03418-15,s3://s3-csu-003/v3-2/Results_10212_25Jun22/
-B6-41,52000,AF-21-05873-20,s3://s3-csu-003/v3-2/Results_10288_25Jun22/
-B6-42,56000,AF-21-05839-18,s3://s3-csu-003/v3-2/Results_10102_28Jun22/
-B6-51,54000,AFT-21-05379-22,s3://s3-csu-003/v3-2/Results_NB501786_0400_25Jun22/
-B6-52,53000,AFT-21-05379-22,s3://s3-csu-003/v3-2/Results_NB501786_0400_25Jun22/
-B6-61,53000,AF-61-00017-19,s3://s3-csu-003/v3-2/Results_10154_28Jun22/
-B6-62,53000,AF-12-00295-18,s3://s3-csu-003/v3-2/Results_10078_28Jun22/
-B6-71,65000,AF-12-00295-18,s3://s3-csu-003/v3-2/Results_10078_28Jun22/
-B6-81,54000,AF-21-05744-19,s3://s3-csu-003/v3-2/Results_10184_28Jun22/
-B6-82,58000,AF-21-05744-19,s3://s3-csu-003/v3-2/Results_10184_28Jun22/
-B6-83,55000,AF-21-06613-17,s3://s3-csu-003/v3-2/Results_10049_27Jun22/
-B6-84,54000,AF-21-05744-19,s3://s3-csu-003/v3-2/Results_10184_28Jun22/
-B6-85,55000,AF-21-05744-19,s3://s3-csu-003/v3-2/Results_10184_28Jun22/
-B6-86,53000,AF-21-05744-19,s3://s3-csu-003/v3-2/Results_10184_28Jun22/
-B6-87,53000,AF-21-05744-19,s3://s3-csu-003/v3-2/Results_10184_28Jun22/
-B6-91,54000,AF-61-00017-19,s3://s3-csu-003/v3-2/Results_10154_28Jun22/
+clade,maxN
+B1-11,56819.399999999994
+B2-11,55194.0
+B3-11,51755.6
+B4-11,53210.85
+B5-11,64171.75
+B6-11,58642.4
+B6-12,51685.45
+B6-13,51968.6
+B6-14,51581.0
+B6-15,51316.0
+B6-16,51211.9
+B6-21,53396.0
+B6-22,51176.0
+B6-23,57180.2
+B6-31,51697.3
+B6-41,51508.4
+B6-42,54087.15
+B6-51,51560.0
+B6-52,51468.4
+B6-61,56273.25
+B6-62,51565.1
+B6-71,66291.75
+B6-81,55315.0
+B6-82,57570.0
+B6-83,52150.7
+B6-84,59318.899999999994
+B6-85,53690.8
+B6-86,55739.5
+B6-87,51724.0
+B6-91,51847.0
```

Comment:
Wonder if it would be neater to round these values. Ncount  will only be an integer